faq/migrate-lucene-to-cosmos.md

```diff
@@ -0,0 +1,75 @@
+---
+description: This article describes how to migrate Lucene Index into Cosmos using SysKit Point Index Migration Tool.
+---
+
+# Migrate Lucene to Cosmos
+
+After you successfully finished **SysKit Point cloud deployment** and you have the **Cosmos database**, you can proceed with the migration of the Lucene Index.
+
+These are the steps:
+
+1. Download the [**SysKit.Point.MigrationTool.exe**](https://downloads.syskit.com/point/files/SysKitPoint-MigrationTool.zip) to the server where the **SysKit Point** is installed.
+2. Inside the **%ProgramData%\SysKit\Point** folder create a new file **migrationConfig.json**.
+3. Modify the file and add the configuration parameters - here is the example of minimal configuration file:
+
```

Comment:
ovaj migrationConfig bi bolje bilo formatirati kao kod sa `{
    "DatabaseId": "PointAudit",
    "TenantGuid": "Microsoft 365 tenant ID",
    "Endpoint": "Cosmos URI",
    "Key": "Cosmos PK",
    "AuditIndexLocation": "audit index location, example: C:\\\ProgramData\\\SysKit\\\Point_DBG\\\CustomAuditIndexName"
  }`
  isto tako, stavio bi samo prave vrijednosti kao example, bez opisa, budući da ispod navodiš gdje naći potrebne podatke

Reply:
endopoint mi je u obliku linka - to bih trebao maknuti? 

a kod se stavi između ovih trostrukih navodnika ``` ``` - ako sam dobro skužio